modules/installation/pages/uyuni-server-setup.adoc

```diff
@@ -0,0 +1,174 @@
+For more information, see xref:uyuni-server-setup.adoc#server-setup-yast[].
+With this address you open the {productname} {webui}.
+
+. In the {webui}, navigate to the menu:Create Organization[Organization Name] field and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Fill in the Account Information fields including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
++
+
+// image::quickstart-mgr-setup-admin1.png[scaledwidth=80%]
+
+You are now presented with the {productname} menu:Home[Overview] page.
+// In the next section you will prepare the server for connecting the first client.
+
+
+== Optional: Synchronizing Products from {scc}
+
+{scc} (SCC) maintains a collection of repositories which contain packages, software and updates for all supported enterprise client systems.
+These repositories are organized into channels each of which provide software specific to a distribution, release, and architecture.
+After synchronizing with SCC clients may receive updates, and be organized into groups and assigned to specific product software channels.
+
+This section covers synchronizing with SCC from the {webui} and adding your first client channel.
+
+[NOTE]
+====
+For Uyuni, synchronizing products from {scc} is optional.
+====
+
+Before you can synchronize software repositories with SCC, you will need to enter organization credentials in {productname}.
+In previous versions, so-called mirror credentials were used instead.
+The organization credentials give you access to the SUSE product downloads.
+You will find your organization credentials in https://scc.suse.com/organization.
+
+Enter your organization credentials in the {productname} {webui}:
+
+
+[[proc-admin-organization-credentials]]
+.Procedure: Entering Organization Credentials
+. In the {susemgr} {webui}, select menu:Main Menu[Admin > Setup Wizard].
+. From the [guimenu]``Setup Wizard`` page select the btn:[Organization Credentials] tab.
+. Click btn:[Add a new credential].
+. In the dialog, enter [guimenu]``Username`` and [guimenu]``Password``, and confirm with btn:[Save].
+
+When the credentials are confirmed with a check-mark icon, proceed with <<proc-quickstart-first-channel-sync>>.
+
+
+[[proc-quickstart-first-channel-sync]]
+.Procedure: Synchronizing with {scc}
+. In the {webui}, navigate to menu:Admin[Setup Wizard].
+
+. From the [guimenu]``Setup Wizard`` page select the btn:[SUSE Products] tab.
+Wait a moment for the products list to populate.
+If you previously registered with {scc} a list of products will populate the table.
+This table lists architecture, channels, and status information.
+For more information, see xref:reference:admin/setup-wizard.adoc[Wizard].
++
+
+image::admin_suse_products.png[scaledwidth=80%]
```

Comment:
Probably should move this image out of the procedure, to avoid breaking it up. Can it go at the end?

Reply:
I think the image belongs here.  Later or earlier it's confusing or useless.  With the "+" connector the procedure will hopefully not break up....